spec/asyncapi.md

```diff
@@ -2278,6 +2231,7 @@ Field Name | Type | Applies To | Description
 <a name="securitySchemeObjectBearerFormat"></a>bearerFormat | `string` | `http` (`"bearer"`) | A hint to the client to identify how the bearer token is formatted.  Bearer tokens are usually generated by an authorization server, so this information is primarily for documentation purposes.
 <a name="securitySchemeFlows"></a>flows | [OAuth Flows Object](#oauthFlowsObject) | `oauth2` | **REQUIRED**. An object containing configuration information for the flow types supported.
 <a name="securitySchemeOpenIdConnectUrl"></a>openIdConnectUrl | `string` | `openIdConnect` | **REQUIRED**. OpenId Connect URL to discover OAuth2 configuration values. This MUST be in the form of an absolute URL.
+<a name="securitySchemeScopes"></a>scopes | [`string`] | `oauth2` \| `openIdConnect` | List of the needed scope names. An empty array means no scopes are needed.
```

Comment:
Thinking about adding the `encryption` key to the Security Scheme Object and removing it from the `type` field. Encryption mechanisms are compatible with all the types listed in `type` after all. Maybe the only one to be left out is `plain`? @derberg @dalelane @char0n @smoya @magicmatatjahu @jonaslagoni

Reply:
I'm not sure I understand what this would end up like...

Do you mean we would have, for example, `encryption: scramSha512`? 

If so, then I'm not sure I would like that, because I don't think that fits well - as to me SCRAM is more about describing the security mechanism (the type of challenge-response that the client has to do with the server) than just describing the SHA encryption bit. 

If not, then I've misunderstood :)

Reply:
Sorry for the poor explanation. No, it would be `encryption: symmetric or `encryption: asymmetric. We now have [`type: symmetricEncryption` and `type: assymetricEncryption`](https://github.com/asyncapi/spec/blob/2a7acb2e2503381648c0ff2caa97588f50edb974/spec/asyncapi.md#fixed-fields-22). They both mean that the message content is encrypted, even if the connection is not. I'm considering moving this to a property because it's not an auth type per se. Actually, you can send an encrypted message over any kind of connection.

I'm actually wondering now if this is being used by anyone 😅 It's not very useful because you only get to know that something is encrypted but it doesn't tell you how it's encrypted so, yeah 🤷  

Anyway, this is not really relevant to this PR but just came to my mind and had to share.

Reply:
Thinking further, I think I'm gonna remove encryption types from 3.0.0 and see if someone complains 😄 If they do, we can add them back on 3.1.0 haha! If not, it means nobody cares.